src/treaty2/index.ts

```
@@ -577,49 +584,66 @@ const createProxy = (
-                            temp.forEach((value, key) => {
-                                // @ts-ignore
-                                data[key] = value
-                            })
+                            case 'application/octet-stream':
+                                data = await response.arrayBuffer()
+                                break
 
-                            break
+                            case 'multipart/form-data':
+                                const temp =
+                                    (await response.formData()) as FormData
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)